src/xdocs/config_header.xml

```diff
@@ -225,7 +225,8 @@ line 14: ^ \*/
           four digit year. Line 5 is an example how to enforce revision
           control keywords in a file header. Lines 12-14 is a template for
           javadoc (line 13 is so complicated to remove conflict with and of
-          javadoc comment).
+          javadoc comment). Lines 7, 9 and 11 will be treated as '^$' and
+          will forcefully expect the line to be empty.
```

Comment:
It will be very useful to users if you put this tiny nuance to `header` description `the required header specified inline. Individual header lines must be separated by the strin.........`.

please also add that explanation to last example `To configure the check to verify that each file starts with the header.`, or create new example. Users like examples more that boring manuals.